DABI.js

```diff
@@ -0,0 +1,677 @@
+{
+    "translatorID": "5cf8bb21-e350-444f-b9b4-f46d9fab7827",
+    "label": "DABI",
+    "creator": "Jens Mittelbach",
+    "target": "^https?://dabi\\.ib\\.hu-berlin\\.de/",
+    "minVersion": "1.0",
+    "maxVersion": "",
+    "priority": 100,
+    "inRepository": true,
+    "translatorType": 4,
+    "browserSupport": "gcsvbi",
+    "lastUpdated": "2015-12-15 18:59:56"
+}
+
+/* FW LINE 57:6869c32952b1 */
```

Comment:
that must have been a misunderstanding: Since you are not using the Framework, you can delete everything from line 15 to (and including) line 504